server/controllers/get-followers.controllers.js

```diff
@@ -83,7 +83,7 @@ const getFollowing = async (req, res) => {
       });
     }
     return res.status(200).json({
-      message: 'Successfully retrieved all users you are following',
+      message: 'You have successfully retrieved all users you are following',
```

Comment:
`You have successfully retrieved all the users you are following`

Reply:
Noted and fixed